requirements.txt

```diff
@@ -41,6 +41,7 @@ setuptools>=65.5.1 # Not a direct dependency, pinned to get a security fix
 tornado==6.3.3 # Not a direct dependency, pinned to get a security fix
 
 # Dev
+black==22.*
```

Comment:
Would it make sense to update dev-tools/requirements.txt with the same wildcard?

Reply:
Definitely, I forgot that place